### ISSUE TITLE
Payouts: User Misery did not receive latest distribution 

### ISSUE THREAD

**Issue by sambacha** [manifoldfinance/support#8]
forum post: https://forums.manifoldfinance.com/t/may-2022-fold-staking-payouts/283/9?u=misery

address: 0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a

https://etherscan.io/token/0xd084944d3c05cd115c09d072b9f44ba3e0e45921?a=0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a

**Comment by sambacha** [comment]
Update: 

https://etherscan.io/token/0x454BD9E2B29EB5963048cC1A8BD6fD44e89899Cb?a=0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a

1,099.991194473081321448 XFOLD

### Notes

1,099.991194473081321448 XFOLD

0 Stablecoin payout, why?

**Comment by miseRyxml** [comment]
@sambacha thank you!

My reference for the payout estimate is a wallet which had roughly the same amount of xfold(~1.1k) in the relevant month and received ~37 $usdc and ~21 $fold
https://etherscan.io/address/0x5d0d83a3d77149694a1aa1ec926e6f60d71484bb#tokentxns

My previous month payout however was ~37 $fold (for 1k $xfold).

Based on these numbers, I estimate that I should have received a payout somewhere between min 20 and max 40 $fold plus ~35-40 USDC.

**Comment by sambacha** [comment]
will double check this today, and have it to you by tomorrow

**Comment by sambacha** [comment]
> @sambacha thank you!
> 
> My reference for the payout estimate is a wallet which had roughly the same amount of xfold(~1.1k) in the relevant month and received ~37 $usdc and ~21 $fold [etherscan.io/address/0x5d0d83a3d77149694a1aa1ec926e6f60d71484bb#tokentxns](https://etherscan.io/address/0x5d0d83a3d77149694a1aa1ec926e6f60d71484bb#tokentxns)
> 
> And a friend just told me he received 2% of his $xfold value on payout day in $usdc and 1.8% in $fold, which would equal around 20 $fold and 275 $usdc.
> 
> Based on these numbers, I estimate that I should have received a payout somewhere around 20-22 $fold plus 37-275 $usdc
> 
> (updated)

You have received stablecoin payment and fold payment, unless you are disputing a different month I do not see any issue. Please respond with more details if I am mistaken, I will leave the ticket open

**Comment by miseRyxml** [comment]
@sambacha No. My address was in the disperse function for the $fold distribution but received 0 $fold. Was not even in any $usdc distribution.
Proof:
![image](https://user-images.githubusercontent.com/106352697/172450568-0c38d069-2e6f-4e57-8e78-6fcdd13cc1b8.png)
![image](https://user-images.githubusercontent.com/106352697/172450838-65e49b3e-bc59-49f3-a4c6-3ea759fc1d9b.png)

If you believe I received any, please try to locate the TX that shows it but I can guarantee there is none.

**Comment by miseRyxml** [comment]
@sambacha this might be the reason for the misunderstanding:
The tx mentioned in my estimate answer of this issue thread was not mine. It was another address that had a similar amount of xfold as I had, but **did** receive a payout. I used it as a reference for my estimate. 

This is my address: 0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a

https://etherscan.io/token/0xd084944d3c05cd115c09d072b9f44ba3e0e45921?a=0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a

**Comment by sambacha** [comment]
> @sambacha this might be the reason for the misunderstanding: The tx mentioned in my estimate answer of this issue thread was not mine. It was another address that had a similar amount of xfold as I had, but **did** receive a payout. I used it as a reference for my estimate.
> 
> This is my address: 0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a
> 
> [etherscan.io/token/0xd084944d3c05cd115c09d072b9f44ba3e0e45921?a=0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a](https://etherscan.io/token/0xd084944d3c05cd115c09d072b9f44ba3e0e45921?a=0x8e26530E1d3Ce2e31322ff12DDf27a65cC2Bd92a)

taking care of this now
